test/0.8.25/vaults/dashboard/dashboard.test.ts

```diff
@@ -966,6 +1030,78 @@ describe("Dashboard", () => {
       expect(await steth.balanceOf(vaultOwner)).to.equal(stethBalanceBefore);
       expect(await wsteth.balanceOf(vaultOwner)).to.equal(wstethBalanceBefore - amount);
     });
+
+    it("succeeds with rebalanced shares - 1 share = 0.5 stETH", async () => {
+      await steth.mock__setTotalShares(ether("1000000"));
+      await steth.mock__setTotalPooledEther(ether("500000"));
+      const sharesToBurn = ether("1");
+      const stethToBurn = sharesToBurn / 2n; // 1 share = 0.5 steth
+
+      const permit = {
+        owner: await vaultOwner.address,
```

Comment:
```suggestion
        owner: vaultOwner.address,
```
Here and in some other places below